sdk/src/quic.rs

```diff
@@ -4,4 +4,5 @@ pub const QUIC_PORT_OFFSET: u16 = 6;
 // give significant improvement or seem to impact stability)
 pub const QUIC_MAX_CONCURRENT_STREAMS: usize = 2048;
 
-pub const QUIC_MAX_TIMEOUT_MS: u32 = 120_000;
```

Comment:
High timeout value was causing the connection failure to take longer to detect. For example, if a node is not running QUIC server, it was taking 2 minutes for the connection request to fail.

These updated values ensure that the connection attempt fails sooner. If the connection is established, the keep alive timer (which is shorter than the idle timer) will keep the connective alive.